### DISCUSSION TITLE
ForceGuildId no longer seems to exist, and other things

### DISCUSSION THREAD

**Discussion by onrtrap** [Chew/JDA-Chewtils#58] [Q&A]
Hello, I've been following your documentation to make my own bot, and I found some issues with the article here: `(https://github.com/Chew/JDA-Chewtils/wiki/SlashCommands-in-JDA-Chewtil`)`  
I tried following it to the letter and found two things: 
1. ForceGuildId doesn't exist in the current version of Chewtils that I'm using (1.24.1)
2. The code given for loading the commands is not working.  It gives the error "No Token was entered."  I made sure to insert my bot's own token into the code, so I'm not quite sure what is the case.
It would be really helpful if I could get these problems solved.  I have a lot of plans for the bot, so I hope you could get back.

**Comment by Chew** [comment]
1) Are you referring to `CommandClientBuilder#forceGuildId`? That should exist, can you show your code?
2) Gonna need to see code for this too.

**Reply by onrtrap** [reply]
![image](https://user-images.githubusercontent.com/23125435/159501180-af41b1bd-b3a4-47d6-adf6-7e976ca95531.png)
So I kind of gave up on it, but just to show you, I dropped forceGuildId into the code.  Just in case you don't use intellij, the red means it can't find the method.  
![image](https://user-images.githubusercontent.com/23125435/159501541-01d6f179-79e7-4dcd-a096-767987fccf90.png)
This is the section that is supposed to add the event listener.  All the code here is taken verbatim from the article.
Just as a sidenote, I downloaded Chewtils 1.24.1, and I actually checked the CommandClientBuilder code to see if forceGuildId was there and I made a mistake somehow.  It would be a bit hard to show it to you, but the method isn't there, as in it literally is not in CommandClientBuilder.  It also doesn't show up here: https://chew.pro/JDA-Chewtils/1.24.1/index.html

**Reply by Chew** [reply]
I'll admit, I should've read my own docs. It's `forceGuildOnly`

**Reply by onrtrap** [reply]
I added the code.  It still isn't working correctly.  I'll give the points where there are issues:
![image](https://user-images.githubusercontent.com/23125435/159709352-ce21c491-9db4-472f-925f-402821f4b865.png)
I heavily stripped down my class that handles SlashCommands, down to specifically the ping command only, just as a test.
![image](https://user-images.githubusercontent.com/23125435/159709605-e0e7ee69-0737-427d-880e-842a4bb99a82.png)
I initialize the class here, and get:
![image](https://user-images.githubusercontent.com/23125435/159710132-5fb1b32a-e853-49b4-9695-7993134f8d28.png)
The error being on the third line there.  The first line is a console test for the correct token and the correct prefix (for the non-slash commands)
This is also a desperation play, but:
![image](https://user-images.githubusercontent.com/23125435/159710406-7259f3db-6d1f-4309-9ce6-9501f0874c35.png)
If you do this, you get:
![image](https://user-images.githubusercontent.com/23125435/159710469-7ef522c2-45c9-4b6e-8a84-777bdbea39f4.png)
I'm assuming the issue is in the JDABuilder.

**Reply by onrtrap** [reply]
Also to explain it better, "token" is the bot's token, and it is the correct one.

**Reply by Chew** [reply]
org.avida doesn't seem to be related to any project I know of, so I don't think it's related. Do you have it as a dependency somewhere?
Also, don't leak your token. Be sure to reset it.

**Reply by onrtrap** [reply]
org.avida is the package for my project.  I also made sure to reset my token.   I added a stacktrace() and found that the issue was that I didn't use setOwnerId(), and that seems to have solved the problem.  I think it would be helpful if the article was altered to include this, in the case that my problem was a problem shared by others.
